po/da.po

```diff
@@ -151,876 +151,879 @@ msgstr "Arrays og for-løkker"
 msgid "Day 1: Afternoon"
 msgstr "Dag 1: Eftermiddag"
 
-#: src/SUMMARY.md:44 src/basic-syntax/variables.md:1
+#: src/SUMMARY.md:44 src/SUMMARY.md:295 src/control-flow.md:1
+msgid "Control Flow"
+msgstr "Forgreninger"
+
+#: src/SUMMARY.md:45 src/control-flow/blocks.md:1
+msgid "Blocks"
+msgstr "Blokke"
+
+#: src/SUMMARY.md:46
+msgid "if expressions"
+msgstr "if udtryk"
+
+#: src/SUMMARY.md:47
+msgid "for expressions"
+msgstr "for udtryk"
+
+#: src/SUMMARY.md:48
+msgid "while expressions"
+msgstr "while udtryk"
+
+#: src/SUMMARY.md:49
+msgid "break & continue"
+msgstr "break og continue"
+
+#: src/SUMMARY.md:50
+msgid "loop expressions"
+msgstr "loop udtryk"
+
+#: src/SUMMARY.md:52 src/basic-syntax/variables.md:1
 msgid "Variables"
 msgstr "Variabler"
 
-#: src/SUMMARY.md:45 src/basic-syntax/type-inference.md:1
+#: src/SUMMARY.md:53 src/basic-syntax/type-inference.md:1
 msgid "Type Inference"
 msgstr "Typeudledning"
 
-#: src/SUMMARY.md:46
+#: src/SUMMARY.md:54
 msgid "static & const"
 msgstr "static & const"
 
-#: src/SUMMARY.md:47 src/basic-syntax/scopes-shadowing.md:1
+#: src/SUMMARY.md:55 src/basic-syntax/scopes-shadowing.md:1
 msgid "Scopes and Shadowing"
 msgstr "Virkefelt og overskyggede variabler"
 
-#: src/SUMMARY.md:48 src/memory-management.md:1
+#: src/SUMMARY.md:56 src/enums.md:1
+msgid "Enums"
+msgstr "Enumerationer"
+
+#: src/SUMMARY.md:57 src/enums/variant-payloads.md:1
+msgid "Variant Payloads"
+msgstr "Nyttelast i varianter"
+
+#: src/SUMMARY.md:58 src/enums/sizes.md:1
+msgid "Enum Sizes"
+msgstr "Størrelse af enumerationer"
+
+#: src/SUMMARY.md:60 src/control-flow/novel.md:1
+msgid "Novel Control Flow"
+msgstr "Nyskabende forgreninger"
```

Comment:
I had to go an look at the English version to see what was meant by 'nyskabende'. It seems we want to talk about control flows not commonly found in other language so how about any of `usædvanlige/unikke/specielle forgreninger`? One thing more thing to keep in mind is that match isn't that unusual now that python has it and it has been in functional languages forever so I'm not sure how well the English version will stand the test of time.

Reply:
> One thing more thing to keep in mind is that match isn't that unusual now that python has it and it has been in functional languages forever so I'm now even sure the English version will stand the test of time.

That is a good point, thanks! @fw-immunant and @djmitche, something to take into account, especially for the v2 of the course.

Reply:
I think I like your suggestion:

```suggestion
msgstr "Usædvanlige forgreninger"
```